tests/php/test-amp-style-sanitizer.php

```diff
@@ -2037,7 +2036,7 @@ public function get_external_stylesheet_data() {
 				],
 				'expected_styles' => [],
 				'expected_errors' => [ AMP_Style_Sanitizer::STYLESHEET_FETCH_ERROR ],
-				'cached_data'    => new CachedResponse(
+				'expected_cached_response'    => new CachedResponse(
```

Comment:
Does `TestCase` map associative arrays to named positional args?

Reply:
In PHP 8 it does.